NOpenType/Renderer.cs

```diff
@@ -179,24 +187,130 @@ private static Point<int> GetMidPoint(Point<int> v1, Point<int> v2)
             return new Point<int>((v1.x + v2.x) / 2, (v1.y + v2.y) / 2);
         }
 
-        private int ToPixels(int funits, int pointSize, int dpi)
+        /// <summary>
+        /// Renders the specified text at the specified X and Y position.
+        /// </summary>
+        /// <param name="x">The x postion in pixels to draw the text.</param>
+        /// <param name="y">The y postion in pixels to draw the text.</param>
+        /// <param name="text">The text.</param>
+        /// <returns>The size of the rendered text in pixels.</returns>
+        public Size Render(int x, int y, string text, TextOptions options)
+        {
+            int scalingFactor = ScalingFactor(options.FontSize);
+            var glyphs = Layout(x, y, text, options).ToList();
+            int top = int.MaxValue;
+            int bottom = int.MinValue;
+            int left = int.MaxValue;
+            int right = int.MinValue;
+
+            foreach (var glyph in glyphs)
+            {
+                RenderGlyph(glyph, scalingFactor);
+
+                if (glyph.left < left)
+                {
+                    left = glyph.left;
+                }
+                if (glyph.right > right)
+                {
+                    right = glyph.right;
+                }
+
+                if (glyph.top < top)
+                {
+                    top = glyph.top;
+                }
+                if (glyph.bottom > bottom)
+                {
+                    bottom = glyph.bottom;
+                }
+            }
+            _rasterizer.Flush();
+
+            return new Size()
+            {
+                Width = ((right - left) * scalingFactor) / Constants.FontToPixelDivisor,
+                Height = ((bottom - top) * scalingFactor) / Constants.FontToPixelDivisor
+            };
```

Comment:
I'm using the fact that it returns the size to enable drawing the bounding box in the CLI.

if it didn't return the size then I would need to call `MeasureText` and `Render` separately causing runs of the `layout` method duplicating that piece of work.